core/arch/arm/include/kernel/static_ta.h

```diff
@@ -40,11 +40,12 @@ struct static_ta_head {
 	TEE_Result (*create_entry_point)(void);
 	void (*destroy_entry_point)(void);
 	TEE_Result (*open_session_entry_point)(uint32_t nParamTypes,
-			TEE_Param pParams[4], void **ppSessionContext);
+					TEE_Param pParams[TEE_NUM_PARAMS],
+					void **ppSessionContext);
```

Comment:
For consistency, use the same indentation as `invoke_command_entry_point` below.